app/helpers/restaurant_helper.rb

```diff
@@ -0,0 +1,5 @@
+module RestaurantHelper
+  def create_dropdown_options
+    @select_options = Restaurant::PERMITTED_CATEGORIES.dup.insert(0, 'All Restaurants')
+  end
+end
```

Comment:
Also, you don't need to store the value in an instance variable. Just call the method from the view and return a value.